addons/message_paid_research/manifest.json

```diff
@@ -0,0 +1,55 @@
+{
+  "api_version": "0.1",
+  "id": "message_paid_research",
+  "name": "Participate in paid research to help make VPN better",
+  "type": "message",
+  "translatable": false,
+  "conditions": {
+    "min_client_version": "2.15.0",
+    "translation_threshold": 0,
+    "locales": ["en"],
+    "javascript": "currencyCheck.js"
+  },
+  "javascript": {
+    "enable": "setDate.js"
+  },
+  "message": {
+    "id": "message_paid_research",
+    "title": "Participate in paid research to help make VPN better",
+    "subtitle": "This study will take place in April and May 2023, and you will receive up to $100 USD (or the equivalent in your local currency) as our thanks for your time and feedback.",
```

Comment:
> (or the equivalent in your local currency)


If this is USD-only, is this part of the sentence necessary?

Reply:
Users in Canada and the UK could receive this message (since targeting is only 'en') and would be potentially accepted (only concern is timezone, which applies to UK), and then be paid the equivalent of 100 USD in their local currency.